packages/web/src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, Alert, AlertIcon, AlertDescription } from '@chakra-ui/react';
+export function ErrorMessage({ message }: { message: any }) {
```

Comment:
For types, make sure to give it what you are wanting in this component to help people when they use it. Otherwise they'll need to come to this component to see how it's used 👍 

```suggestion
export function ErrorMessage({ message }: { message: string }) {
```